src/components/SearchQueryFetch/SolrQueryFetchConnector.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import SearchForm from '../SearchForm/SearchForm'
+
+class SolrConnectorDemo extends React.Component {
+  // @ts-ignore
+  constructor(props) {
+    super(props);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.state = {
+      solrSearchUrl: "http://drupalvm.corporate:8983/solr/d7/select",
+      query: "memory",
+      filter: "",
+      fetchFields: ""
+    }
+  }
+
+  // @ts-ignore
+  onSubmit(event) {
+    event.preventDefault();
+    let searchParams = {
+      solrSearchUrl: "http://drupalvm.corporate:8983/solr/d7/select",
+      // @ts-ignore
+      query: this.state.query,
+      // @ts-ignore
+      filter: [this.state.filter],
+      // @ts-ignore
+      fetchFields: this.state.fetchFields.split(" "),
+      offset: 0,
+      limit: 10,
+      facet: {
+        item_bundle: {
+          type: "terms",
+          field: "ss_item_bundle",
+          limit: 20
+        },
+        topics: {
+          type: "terms",
+          field: "ss_node$type",
+          limit: 20
+        },
+      },
+      highlightParams: {
+        "hl": "true",
```

Comment:
use full names instead of abbreviations to make each property more readable

Reply:
This reflects the properties coming from solr.

Reply:
ah ok